src/preflight.c

```diff
@@ -239,14 +240,14 @@ static void* preflight_worker_handle_device_add(void* userdata)
 	plist_free(value);
 
 	if (!platform_str) {
```

Comment:
Also, can you please rename all occurrences of `platform_str` to something like `device_class`, thanks :)

Reply:
done, that was just me lazy 😸